webpack/webpack.common.js

```diff
@@ -40,18 +40,29 @@ module.exports = {
         ]
       },
       {
-        test: /\.svg$/,
-        oneOf: [
-          {
-            // load svg modules to be displayed as icons/images
-            loader: 'svg-react-loader'
-          },
+        test: /\.svg$/i,
+        use: [
           {
-            // load svg files normally (eg. fonts, etc)
-            loader: 'url-loader?limit=100000'
+            loader: 'url-loader',
+            options: {
+              encoding: false
+            }
           }
         ]
       }
+      // {
```

Comment:
if you don't need it  - you can remove it

Reply:
removed